test/data/StorageFileTest/ValidData.xml

```diff
@@ -4,13 +4,13 @@
         <name>John Doe</name>
         <phone isPrivate="false">98765432</phone>
         <email isPrivate="false">johnd@gmail.com</email>
-        <address isPrivate="false">John street, block 123, #01-01</address>
+        <address isPrivate="false">123, John street, #01-01, 000000</address>
```

Comment:
Not sure why these tests (and all of the subsequent tests) needed to be changed to include additional information in the address. Is this relevant to your change?